src/index.ts

```diff
@@ -4,6 +4,8 @@ import QrcodeStream from './components/QrcodeStream.vue'
 import QrcodeCapture from './components/QrcodeCapture.vue'
 import QrcodeDropZone from './components/QrcodeDropZone.vue'
 
+import { setZXingModuleOverrides } from '@sec-ant/barcode-detector'
```

Comment:
This import is [a combination of `pure` and `side-effects`](https://github.com/Sec-ant/barcode-detector#both), so it can potentially pollute `globalThis` unintendedly. If only `pure` module is needed, as shown in the changes of `scanner.ts`, I suggest changing the import path here to `@sec-ant/barcode-detector/pure`, too :)